www/docs/en/dev/guide/support/index.md

```diff
@@ -89,6 +94,7 @@ the core plugins, additional APIs are available via
         <td data-col="ios"     class="y"></td>
         <td data-col="osx"     class="n"></td>
         <td data-col="windows" class="y">Windows Phone 8.1 only</td>
+        <td data-col="electron" class="p">Tests Pending</td>
```

Comment:
What does this mean? Either it is known working, or not.

Reply:
These plugins were not tested. I was wondering maybe the user's should know the actual situation, so I just put "Tests Pending". Well in the case, where is either working or not, I can just say it's not working for now.